spec/support/webmock/court_data_api/webmock.rb

```diff
@@ -62,6 +62,16 @@
     )
   end
 
+  config.before(:each, :stub_appeals_defendants_case_search) do
```

Comment:
You don't actually need this. If you delete it, and the reference to it in the spec file, the specs still pass fine, because the `describe` block at the top of the file already has the `stub_defendants_case_search` which is identical to this.